### PR TITLE
New post: Goodbye Summer Ruby Meetup Sept 2023 recap

### PR DESCRIPTION
**Pull request by jurgenwerk**
In progress

### CODE REVIEW

**Comment by jurgenwerk** [comment]
@MrChriss to provide photos and videos

**Comment by miharekar** [comment]
`jekyll serve --watch` doesn't work anymore?

File: _posts/2023-09-28-september-meetup.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Andraž "Brodul" Brodnik had a talk about deployment of Ruby applications in AWS ECS (Elastic Container Service). In AWS jargon, he guided us through the process of making an application accessible to the public, explaining common pitfalls and providing comparisons to alternatives. The discussion also delved into deployment strategies, particularly the blue-green approach, which piqued the audience's interest.
+
+Next was Tomaž Zaman, with a witty talk named "Ruby is boring tech and why that's good", where he delved into his entrepreneurial journey co-founding Codeable and why he chose Ruby on Rails as the tech basis of the business. His presentation included anecdotes interwoven with humor and hard-earned wisdom.
+
+There was also a mini presentation, by the legendary Miha Rekar. It was heartening to see someone still with rubies in his eyes, even after many years of working with Ruby. With enthusiasm, he showcased the latest features in Rails 7.1 and how he incorporates them into his projects.
```
Comment (miharekar) [inline _posts/2023-09-28-september-meetup.md:14]: "legendary" 🤣

**Comment by jurgenwerk** [comment]
> `jekyll serve --watch` doesn't work anymore?

It didn't work for me so I followed what people did in https://github.com/jekyll/jekyll/issues/8523 (it includes the error I was experiencing)

**Comment by miharekar** [comment]
@jurgenwerk ah, of course. Makes sense. I've updated master, bumped the gems while at it, and rebased this.

**Comment by MrChriss** [comment]
@jurgenwerk sent pictures and video links your way. 

Thank you for writing the recap! :pray: :raised_hands:

**Comment by jurgenwerk** [comment]
https://www.rug.si/2023/09/28/september-meetup/

**Pull request merged**